usaspending_api/accounts/tests/test_tas_autocomplete.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+import pytest
+from model_mommy import mommy
+from rest_framework import status
+
+from usaspending_api.accounts.models import TreasuryAppropriationAccount
+from usaspending_api.common.tests.autocomplete import check_autocomplete
+
+
+@pytest.fixture
+def tas_data(db):
+    mommy.make(
+        TreasuryAppropriationAccount,
+        tas_rendering_label='zzz',
```

Comment:
I just copy and pasted most of these :P Someone else came up with the names first.